SPECS/containerd2/containerd2.spec

```diff
@@ -28,17 +31,33 @@ Requires: runc >= 1.2.1
 
 %description
 containerd is an industry-standard container runtime with an emphasis on
-simplicity, robustness and portability. It is available as a daemon for Linux
+simplicity, robustness, and portability. It is available as a daemon for Linux
 and Windows, which can manage the complete container lifecycle of its host
 system: image transfer and storage, container execution and supervision,
 low-level storage and network attachments, etc.
 
 containerd is designed to be embedded into a larger system, rather than being
 used directly by developers or end-users.
 
+%package -n moby-containerd-cc
+Summary: Industry-standard container runtime with Kata Containers support
+Group: Tools/Container
+Provides:  moby-containerd-cc = %{version}-%{release}
+Conflicts: %{upstream_name}2
+Obsoletes: %{upstream_name}2 < %{version}-%{release}
+Requires:  %{name} = %{version}-%{release}
+
+%description -n moby-containerd-cc
+This flavor of containerd includes additional patches to support Kata Containers features.
+
 %prep
 %autosetup -p1 -n %{upstream_name}-%{version}
 
+# Apply moby-containerd-cc patches conditionally
+%if "%{name}" == "moby-containerd-cc"
```

Comment:
Can we remove the `moby` to stay aligned with the new containerd naming for 3.0?

Reply:
Updated to be a single package with single source (no-cc package anymore).